nixos/doc/manual/installation/installing.xml

```diff
@@ -42,7 +42,7 @@
   </para>
 
   <para>
-   If the text is too small to be legible, try <command>setfont ter-132n</command>
+   If the text is too small to be legible, try <command>setfont ter-v32n</command>
```

Comment:
Should we just detect larger tty's in the installer and scale up automatically?

Reply:
Anyway. This should not be part of this PR.

Reply:
Is there an existing issue/PR for making nixos-minimal iso autodetect hidpi and choose a larger font?